WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -87,5 +87,6 @@ private extension OrderTableViewCell {
         titleLabel.applyHeadlineStyle()
         totalLabel.applyBodyStyle()
         paymentStatusLabel.applyFootnoteStyle()
+        paymentStatusLabel.numberOfLines = 0
```

Comment:
My first instinct would be moving this to PaddedLabel itself. But it seems like we use that label in several places, so I am not sure how safe that would be.

Reply:
good question - I think for labels, sometimes they don't need to be fully shown if the information is not critical and if the vertical space is limited. While for buttons, since users interact with them, it might make more sense to show full text

Reply:
For the Order List, I would make the argument that a max fixed font size set in this cell would be a better way to go. 
Why?
1. Since we're limited on vertical space, it breaks the readability if the word breaks to a new line.
2. Since the payment status is also inside the Order Details screen, where it has plenty of vertical and horizontal space, the user has the opportunity to read it comfortably and in an accessible manner.

"Completed" in this context is almost unreadable, because it's unnatural to break a single word into multiple lines: 
<img src="https://user-images.githubusercontent.com/1062444/60469424-3693fc80-9c22-11e9-919e-bbc699431d7b.png" width="350" />

But its fully rendered size in Order Details
<img src="https://user-images.githubusercontent.com/1062444/60469510-8a064a80-9c22-11e9-9ead-119ac71391ba.png" width="350" />

Reply:
> For the Order List, I would make the argument that a max fixed font size set in this cell would be a better way to go.

Yea with very large font sizes, the price label dominates the horizontal space. Were you suggesting capping the font size of some of the labels in the Order cell? If so, maybe can use this [`scaledFont(for:maximumPointSize:)`](https://developer.apple.com/documentation/uikit/uifontmetrics/2877383-scaledfont) with `UIFontMetrics`?

Reply:
>Were you suggesting capping the font size of some of the labels in the Order cell?

Yes, exactly. I did ask in Slack p1562017670132500-slack-C02Q9GACF what the other iOS devs have done in this situation before. I don't think this exact situation has occurred before, but something very similar. I agree with @ctarda that we should get @kyleaparker's opinion as well.

Reply:
Just to add a note to this, you might want to look at `fontForTextStyle(style:weight:maximumPointSize)` in `DefaultStyle`, which should provide what you want